tests/ui/circ_policies/test_circ_policies_api.py

```diff
@@ -34,7 +35,15 @@ def test_no_default_policy(app):
     assert not cipo
 
 
-def test_circ_policy_create(db, circ_policy_martigny_data_tmp):
+def test_circ_policy_create(circ_policy_martigny_data_tmp,
```

Comment:
This function tests only an validation error during creation, maybe add successful creation or change function name.

Reply:
I dont think so. This PR add validation error check but existing  tests was already there.